main.js

```diff
@@ -192,4 +192,26 @@ btns.forEach((btn, index) => {
   btn.addEventListener('click', () => {
     popUp(index);
   });
-});
\ No newline at end of file
+});
+
+// eslint-disable-next-line array-bracket-spacing, comma-dangle
+const [form, ] = document.getElementsByClassName('contactform');
+const error = document.querySelector('.error');
+const validEmailRegExp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+// eslint-disable-next-line no-unused-vars
+const message = document.getElementById('message');
+form.addEventListener('submit', (e) => {
+  // eslint-disable-next-line no-unused-vars
+  const { name, email } = form.elements;
+  const message = [];
```

Comment:
- [ ] for clean code and best JS practices, kindly remove the disable ESLint comments and fix the problems as they might affect your code ahead. 😄

Reply:
I appreciate the review. I've made the changes requested.